app/api/items/get_item.go

```diff
@@ -13,8 +13,9 @@ import (
 type itemStringCommon struct {
 	// required: true
 	LanguageID int64 `json:"language_id,string"`
+	// Nullable
```

Comment:
You haven't modified the add/update items where the title could be "null" as well then :-/ (even I don't really know what it means to have a `null` title)

Reply:
FYI, it would mean that there is no translation for the title in this language and that the default language should be used. (no impact for us)

Reply:
> You haven't modified the add/update items where the title could be "null" as well then :-/ (even I don't really know what it means to have a `null` title)

I haven't modified add/update intentionally because we don't allow setting titles to NULL there. Should we (I doubt we want to allow that)?

Reply:
Ok, valid reason :-)